Makefile

```diff
@@ -144,9 +144,6 @@ $(ARRAI_OUT)/% : codegen/testdata/%/sysl.json $(ARRAI_TRANSFORMS)/*.arrai
 
 arrai: $(patsubst %,codegen/arrai/tests/%,$(targets))
 
-.PHONY: arrai-nodiff
-arrai-nodiff: $(patsubst %,codegen/arrai/tests/%.nodiff,$(targets))
-
 codegen/arrai/tests/%.nodiff: $(ARRAI_OUT)/% $(TIDY)
```

Comment:
Remove this bit also please:
```
codegen/arrai/tests/%.nodiff: $(ARRAI_OUT)/% $(TIDY)
    # Ignore dbendpoints, since it's not generated by Sysl transforms.
    # TODO: Remove this check altogether once Sysl transforms are replaced.
```